src/radio.scss

```diff
@@ -47,6 +49,11 @@ $block: #{$fd-namespace}-radio;
 
   @extend %fd-radio-input-hidden;
 
+  &__wrapper {
```

Comment:
![image](https://user-images.githubusercontent.com/6586561/118261961-e7c17a00-b4bc-11eb-8b1e-ff0290a8f51b.png)
Same issue repeating with a checkbox. @droshev should it be fixed as a separate issue?

Reply:
@N1XUS I think so, checkbox is different component right?